app/pages/organization/stats/components/ProjectStats/helpers/getCompleteness.js

```diff
@@ -0,0 +1,20 @@
+function getCompleteness(resource) {
+  let { completeness } = resource;
+
+  if (completeness !== 1
+    && resource.configuration
+    && resource.configuration.stats_completeness_type
+    && resource.configuration.stats_completeness_type === 'classification'
+    && resource.subjects_count
+    && resource.retirement
+    && resource.retirement.options
+    && resource.retirement.options.count
+    && resource.classifications_count) {
+    const totalCount = resource.subjects_count * resource.retirement.options.count;
```

Comment:
Ah, interesting. I didn't know so much went into finding an accurate completeness count. What does this `totalCount` represent exactly? This is multiplying the total number of subjects by the number of retired subjects? I'm not exactly sure what `resource.retirement.options.count` represents.

Reply:
`resource.retirement.options.count` represents the workflows retirement limit for a subject, e.g. retire at 15 classifications and I have 10 subject == 150 total classifications to reach the goal of the workflow. 

Using a metric like this allows progress to be cumulative and not lagging on the stats page, see this issue for a discussion https://github.com/zooniverse/Panoptes/issues/2492